components/widget/widget_table_cell.lua

```diff
@@ -0,0 +1,47 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Widget/Table/Cell
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Widget = require('Module:Infobox/Widget')
+
+local TableCell = Class.new(
+	Widget,
+	function(self, input)
+		self.content = input.content or {}
+		self.classes = input.classes or {}
+		self.css = input.css or {}
+	end
+)
+
+function TableCell:addContent(text)
+	table.insert(self.content, text)
+	return self
+end
+
+function TableCell:addClass(class)
+	table.insert(self.classes, class)
+	return self
+end
+
+function TableCell:make()
+	local cell = mw.html.create('div')
+	cell:css('border-left', '1px solid #bbb'):css('border-top', '1px solid #bbb'):css('background', 'inherit')
+	cell:css('padding', '5px'):css('line-height', '1.42857143')
```

Comment:
Why 5px padding? Is that legacy?

Reply:
Looks very cramped without it 
![image](https://user-images.githubusercontent.com/3426850/174560443-d4a9ab89-8243-4952-946e-66aa43819013.png)

Reply:
I'm sure, but the value is a bit weird. Usually you use multiples of two, e.g. 2 -> 4 -> 8 -> 12 -> 16

Reply:
Ah true, it's legacy. Can swap to 4px